tests/ounit/lustre/lustreSyntaxChecks/misplaced_node_item_frame.lus

```diff
@@ -0,0 +1,18 @@
+node test(x: int) returns(y1, y2: int);
+let
+   check (y1 = 2 or y1 = 3);
+
+   y1 = 2;
+
+   frame ( y2 )
+   y2 = 3;
+   let
+      assert x > 0;
+      y2 = pre x;
+   tel
+tel;
+
+node oscillate() returns (Z: bool)
+let
+   Z = true -> not pre Z;
+tel;
```

Comment:
Kind 2 returns the following error for this input: `Node item assert (x > 0); is not allowed in if block`

According to a previous discussion, Kind 2 should report:
`Asserts are not allowed in if block`